content/influxdb/cloud/reference/release-notes/cloud-updates.md

```diff
@@ -11,7 +11,80 @@ aliases:
   - /influxdb/cloud/reference/release-notes/influxdb-cloud
 ---
 
-InfluxDB Cloud updates occur frequently. Find a compilation of monthly updates below.
+InfluxDB Cloud updates occur frequently. Find a compilation of recent updates below.
```

Comment:
With this change, we may want to change the title/name of the doc in the front matter:

```diff
- title: InfluxDB Cloud monthly updates
+ title: InfluxDB Cloud updates
...
-     name: Monthly updates
+     name: InfluxDB Cloud updates
```

Reply:
makes sense